libs/vendored/jwtool/project.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "jwtool",
+  "$schema": "../../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/vendored/jwtool/src",
+  "projectType": "library",
+  "tags": [],
+  "targets": {
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/libs/vendored/jwtool",
+        "tsConfig": "libs/vendored/jwtool/tsconfig.lib.json",
+        "packageJson": "libs/vendored/jwtool/package.json",
+        "main": "libs/vendored/jwtool/src/index.ts",
+        "assets": ["libs/vendored/jwtool/*.md"]
+      }
+    },
+    "test-unit": {
+      "executor": "@nx/jest:jest",
+      "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
+      "options": {
+        "jestConfig": "libs/vendored/jwtool/jest.config.ts",
+        "testPathPattern": ["^(?!.*\\.in\\.spec\\.ts$).*$"]
```

Comment:
Mostly unrelated, but fun weekend conversation&hellip; I can't remember where we discussed this earlier (and if it was @StaberindeZA or @julianpoy) but is the `.in.spec.ts` files because they are integration tests, so this is using some negative lookahead to find unit test files that don't have the `*.in.*` slug?

Would it be easier/clearer to have `*.in.spec.ts` and `*.unit.spec.ts` to make the greps easier?